src/apps/jewelswap/styled/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Card } from '@mui/material';
+import { Text } from 'src/components/StyledComponents/StyledComponents';
+import styled from 'styled-components';
+
+export const NFTMarketplaceCard = styled(Card)(({ theme }) => ({
+  '&&&': {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+    padding: '1rem',
+    borderRadius: '10px',
+    boxShadow: 'none',
+    transition: 'all .2s linear',
+    minWidth: '250px',
+    backgroundColor: theme.palette.background.secondary,
+  },
+}));
+
+export const NFTMarketplaceImgContainer = styled(Box)(({ theme }) => ({
+  '&&&': {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    background: '#14131C',
+    color: theme.palette.text.homeCards,
+    borderRadius: '10px',
+    minHeight: '125px',
+  },
+}));
+
+export const NFTMarketplaceDescription = styled(Text)(({ theme }) => ({
+  '&&&': {
+    color: theme.palette.text.homeCards,
+    fontWeight: 500,
+    textAlign: 'justify',
+  },
+}));
+
+export const ImageText = styled(Text)(({ theme }) => ({
+  '&&&': {
+    color: theme.palette.text.homeCards,
+    fontWeight: 500,
+    textAlign: 'justify',
+  },
+}));
```

Comment:
These should be renamed to express the domain where there are used. Otherwise, if thry the same as from our app, we can rename those and avoid code duplication